projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -70,8 +88,20 @@ export class TabParagraphContainerComponent
     )
   );
 
-  select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+  select(tabNum: number, event?: Event): void {
+    this.breakpointService
+      .isDown(BREAKPOINT.sm)
+      .pipe(take(1))
+      .subscribe((res) => {
+        if (res) {
+          this.activeTabNum = this.activeTabNum === tabNum ? -1 : tabNum;
+          if (event && event['path']) {
+            window.scrollTo(0, event['path'][1].offsetTop);
```

Comment:
Using global `window` object will break SSR. Please use introduced the `winRef` safe wrapper instead.